client/post-editor/media-modal/detail/index.jsx

```diff
@@ -16,7 +16,7 @@ var DetailItem = require( './detail-item' ),
 import { ModalViews } from 'state/ui/media-modal/constants';
 import { setEditorMediaModalView } from 'state/ui/editor/actions';
 
-const EditorMediaModalDetail = React.createClass( {
+export const EditorMediaModalDetail = React.createClass( {
```

Comment:
This breaks the detail view in the editor, since [the editor modal imports this file via CommonJS](https://github.com/Automattic/wp-calypso/blob/cf2a0b128e0525a5dbfdf241f5b594f34c02c325/client/post-editor/media-modal/index.jsx#L18). When exporting both default and named exports, we must either change all consuming modules to import via ES6 imports, or reference the `.default` or `.EditorMediaModalDetail` export.

Ref: p1477925766002698-slack-calypso-framework